contracts/src/v0.8/l2ep/test/v1_0_0/shared/BaseValidator.t.sol

```diff
@@ -0,0 +1,56 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import {BaseValidator} from "../../../dev/base/BaseValidator.sol";
+import {MockBaseValidator} from "../../mocks/MockBaseValidator.sol";
+import {L2EPTest} from "../L2EPTest.t.sol";
+
+contract BaseValidatorTest is L2EPTest {
+  address internal constant L2_SEQ_STATUS_RECORDER_ADDRESS = address(0x491B1dDA0A8fa069bbC1125133A975BF4e85a91b);
```

Comment:
Are these addresses random? If so maybe nice to use `makeAddr("L2_SEQ_STATUS_RECORDER_ADDRESS");` to clearly indicate we don't care about the actual value

Reply:
looks much better! updated